pkg/api/v1/config.go

```diff
@@ -33,6 +34,14 @@ func loadConfig(filename string) (*AppConfig, error) {
 	return &appConfig, nil
 }
 
+func IsClowderEnabled() bool {
+	enabledValue, err := strconv.ParseBool(os.Getenv("CLOWDER_ENABLED"))
```

Comment:
Why not ACG_CONFIG?

Reply:
Well - because we had kinda standardized on CLOWDER_ENABLED - I think this all ties back to the fact that originally the client was designed to fall over if we didn't have the ACG_CONFIG - now we just nil return, so we could slowly migrate away from using CLOWDER_ENABLED and just use the ACG_CONFIG. If you prefer this way - let me know and I'll change it